kedro-airflow/RELEASE.md

```diff
@@ -1,3 +1,8 @@
+# Release 0.5.1
+* Add additional cli argument `--jinja-file` to provide custom Jinja2 Templates
+* Change the `--target-dir` to `--target-path` so a users can define directory and filename of the output
+* Change e2e behave tests to adjust for changes in pandas-iris starter (no more train node and loading from parameters instead of params:)
```

Comment:
```suggestion
```
We don't typically put changes to things that aren't user-facing in the changelog.